### PR TITLE
Build: Use file-relative path when creating CSS.

### PR DESCRIPTION
**Pull request by simison**
Follow-up from #30473

I've simply picked a commit @jeherve did to fix issues that didn't completely get fixed by #30473

Convo p1548872848551500-slack-jetpack-gutenberg

## Testing
gutenpack-jn

### CODE REVIEW

**Comment by simison** [comment]
Apparently still not fixed by this PR.

You can replicate with:
```bash
rm -f public/custom-properties.css

npm run sdk -- gutenberg client/gutenberg/extensions/presets/jetpack
```

**Comment by blowery** [comment]
Hmmm... custom-properties.css is built two ways: via an npm script or via a webpack plugin. The plugin is using relative file paths, perhaps that's breaking for the way the sdk is being called?

File: package.json [hunk 1]
```diff
@@ -243,7 +243,7 @@
     "build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || npm run build-client",
     "build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || npm run build-client",
     "build-packages": "node bin/build-packages.js",
-    "sdk": "node bin/sdk-cli.js",
+    "sdk": "npm run prebuild-css && node bin/sdk-cli.js",
```
Comment (simison) [inline package.json]: So this fixes it for `npm run sdk` but it doesn't solve the problem if `node bin/sdk-cli.js` is ran directly (like it is Circle CI)

**Comment by blowery** [comment]
yeah, the execSync line is busted. Fixing.

**Review by simison — COMMENTED** [review]
Works!

Tested running the SDK build as well this:

```bash
cd client/gutenberg/extensions/presets/jetpack
npm publish --dry-run
```

**Review by lynnbgriffith — APPROVED** [review]
LGTM

**Pull request merged**